ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlRestApplicationTest.java

```diff
@@ -126,6 +128,10 @@
   private String logCreateStatement;
   private KsqlRestApplication app;
   private KsqlRestConfig restConfig;
+  private KsqlSecurityContext securityContext;
+
+  private ArgumentCaptor<KsqlSecurityContext> securityContextArgumentCaptor =
+      ArgumentCaptor.forClass(KsqlSecurityContext.class);
```

Comment:
This captor is used to verify a `KsqlSecurityContext` contain the correct `ServiceContext` when running the tests.